HelpSource/Help.schelp

```diff
@@ -2,38 +2,266 @@ TITLE::Help
-## link::Browse:: || Browse all documents by categories
+## soft:: ::• soft:: ::link::Classes/Server:: – a real-time audio engine.||
+The default server is link::Classes/Server#Switching the server application#scsynth::. To take full advantage of multi-core, an alternative sound engine, link::Classes/Server#Switching the server application#supernova::, can be used.
+
+## soft:: ::• soft:: ::Client soft:: ::– sclang. a text-based programming language and interpreter. ||
+
+emphasis::sclang:: controls audio synthesis processes on the server and offers an link::Overviews/JITLib##interactive programming environment for on-the-fly coding::, link::Tutorials/A-Practical-Guide/PG_01_Introduction##an extensive library for creating patterns:: and link::Classes/Scale##a library of scales and modes from various cultures::.
+link::Tutorials/Getting-Started/15-Sequencing-with-Routines-and-Tasks##Creating processes that unfold over time:: is straighforward in sclang.
+It also provides link::Guides/GUI-Introduction##all the essential features of a graphical user interface:: that can be used to create custom user interfaces and data visualization.
+With support for
+link::Guides/UsingMIDI##MIDI::,
+link::Guides/OSC_communication##OSC:: and
+link::Guides/Working_with_HID##HID::,
+sclang can be easily interfaced with local and networked hardware and software.
+
+## soft:: ::• soft:: ::link::Classes/ScIDE:: soft:: ::– a dedicated editor for SuperCollider with an integrated help system.||
+link::Guides/SCIde##The SuperCollider-IDE (Integrated Development Environment) :: allows you to start writing code right out of the box. Community members also maintain sclang integration with other popular IDEs such as link::https://github.com/scztt/vscode-supercollider##Visual Studio Code, VSCodium::, link::https://github.com/capital-G/sc_kernel##Jupyter::, link::https://github.com/davidgranstrom/scnvim##SCNvim::, link::https://github.com/supercollider/scvim##scvim::, link::https://github.com/supercollider/scel##Emacs::, and so on.
+::
+
+link::Guides/ClientVsServer##The server and client:: communicate via link::http://opensoundcontrol.org/##OSC (open sound control):: messages over link::Guides/Glossary#kw_udp#UDP::, allowing SuperCollider to run on a single machine or link::Guides/MultiClient_Setups##on multiple machines over a network::. Users can also diretly control the sc-server using link::Guides/NodeMessaging::.
+
+If you wish, you can extend SuperCollider's functionality by installing optional link::https://github.com/supercollider/sc3-plugins##sc3-plugins:: distributed by the SC development team, and by installing individual extensions from the custom libraries called link::Classes/Quarks:: developed by individual users. For more information, see the link::https://github.com/supercollider-quarks/quarks##Directory of community contributed Quarks for SuperCollider:: and the link::https://quarks.supercollider.online/##SuperCollider Quarks Web Index::.
```

Comment:
```suggestion
If you wish, you can extend SuperColliders functionality by installing optional link::https://github.com/supercollider/sc3-plugins##sc3-plugins:: distributed by the SC development team, and by installing individual extensions from the custom libraries called link::Classes/Quarks:: developed by individual users. For more information, see the link::https://github.com/supercollider-quarks/quarks##Directory of community contributed Quarks for SuperCollider:: and link::https://baryon.supercollider.online/##Baryon::, a package index for SuperCollider Quarks.
```

Reply:
quarks.supercollider.online is deprecated in favor of baryon :)

Reply:
"SuperCollider's" was correct

Reply:
my bad, sorry :D